java/src/main/java/vowpalWabbit/learner/VWBase.java

```diff
@@ -63,6 +64,23 @@ final boolean isOpen() {
         return isOpen;
     }
 
+    /**
+     * Save the model in the VW instance.
+     */
+    public void saveModel(File filename) {
+        lock.lock();
+        try {
+            if (isOpen()) {
+                VWLearners.saveModel(nativePointer, filename.getPath());
```

Comment:
It can handle relative paths in its current form actually - the underlying C++ interprets them correctly.